Code/autopkglib/URLDownloader.py

```diff
@@ -162,7 +162,7 @@ def main(self):
         # this can cause issues if this item is eventually copied to a Munki repo
         # with the same permissions and the file is inaccessible by (for example)
         # the webserver.
-        os.chmod(pathname_temporary, 0644)
+        os.chmod(pathname_temporary, "0644")
```

Comment:
This looks wrong.

Reply:
`0o644` might be a better change if the goal is Python3-compatibility.

Reply:
You're totally right, let's use the right octal.